prow/gerrit/adapter/adapter.go

```diff
@@ -303,7 +303,7 @@ func (c *Controller) Sync() {
 		if len(changes) == 0 {
 			return
 		}
-		log.Infof("Processing %d changes for instance: %s", len(changes), instance)
+		log.WithFields(logrus.Fields{"Instance": instance, "Changes": len(changes)}).Infof("Finished querying instance for changes")
```

Comment:
So far all other logs are lowercase words connected by underscores I think
```suggestion
		log.WithFields(logrus.Fields{"instance": instance, "changes": len(changes)}).Infof("Finished querying instance for changes")
```

Reply:
Done